src/modules/feedback/i18n/feedback.provider.js

```diff
@@ -0,0 +1,36 @@
+export const provide = (lang) => {
+	switch (lang) {
+		case 'en':
+			return {
+				feedback_header: 'Map Feedback',
+				feedback_could_not_save: 'Feedback could not be saved',
+				feedback_saved_successfully: 'Feedback saved',
+				feedback_markChangeNotice: 'Markierung Ihrer Änderungsmeldung',
+				feedback_categorySelection: 'Auswahl der Kategorie',
+				feedback_changeDescription: 'Beschreibung der Änderung',
+				feedback_eMail: 'eMail',
+				feedback_privacyPolicy: 'Hinweis zum Datenschutz',
+				feedback_pleaseSelect: 'Please select at least one Point',
+				feedback_disclaimer:
+					'Das LDBV behält sich grundsätzlich vor, Meldungen nicht zu übernehmen. Für evtl. Rückfragen, sowie zur Information über die weitere Bearbeitung, empfehlen wir die Angabe Ihrer E-Mail-Adresse.'
```

Comment:
In some cases, the LDBV cannot adopt your feedback. For questions relating to your feedback and to keep you informed about the work in progress we recommend submitting your email address as well.